packages/shared/src/lib/featureManagement.ts

```diff
@@ -33,10 +34,10 @@ const feature = {
   onboardingChecklist: new Feature('onboarding_checklist', true),
   showCodeSnippets: new Feature('show_code_snippets', false),
   pricingIds: new Feature('pricing_ids', {
-    pri_01jdkx6sakk5kb6p586nejnqam: 'monthly',
-    pri_01jbsccbdbcwyhdy8hy3c2etyn: 'monthly',
-    pri_01jjvm32ygwb1ja7w52e668fr2: 'yearly',
-    pri_01jbscda57910yvwjtyapnrrzc: 'yearly',
+    pri_01jkzj0n2jf89kts09xztfr47b: PlusPriceType.Yearly, // Annual discounted
+    pri_01jbscda57910yvwjtyapnrrzc: PlusPriceType.Yearly, // Annual
+    pri_01jbsccbdbcwyhdy8hy3c2etyn: PlusPriceType.Monthly, // Monthly
+    pri_01jjvm32ygwb1ja7w52e668fr2: PlusPriceType.Yearly, // One-Year Gift
   }),
```

Comment:
Why did these change, are they the sandbox ones?

Reply:
Those are all production values, mainly I've sorted and removed the monthly discounted price and added the Yearly special.